CCContinuousGlucose/Classes/ContinuousGlucoseMeasurement.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  ContinuousGlucoseMeasurement.swift
+//  Pods
+//
+//  Created by Kevin Tallevi on 4/19/17.
+//
+// https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.characteristic.cgm_measurement.xml
+
+import Foundation
+import CCToolbox
+
+public class ContinuousGlucoseMeasurement : NSObject {
+    public var packetSize: UInt16 = 0
+    public var cgmTrendInformationPresent: Bool?
+    public var cgmQualityPresent: Bool?
+    public var sensorStatusAnnunciationFieldWarningOctetPresent: Bool?
```

Comment:
The CGM sensor may not support quality, trend information etc, which it indicates in the features characteristic.